web/src/app/shared/components/nav-bar/nav-bar.component.ts

```diff
@@ -8,19 +8,24 @@ import { AuthService } from '@shared/services/auth/auth.service';
 })
 export class NavBarComponent implements OnInit {
   currentUser: any;
+  isAdmin: any;
   userName: string;
+
   constructor(public authService: AuthService) {
     this.getUser();
-   }
+  }
 
   ngOnInit() {
   }
+
   async getUser() {
     this.currentUser = await this.authService.getCurrentUser();
     if ( this.currentUser ) {
       this.userName = this.currentUser.given_name + ' ' + this.currentUser.family_name;
+      this.isAdmin = this.currentUser.admin === '1' ? true : false;
```

Comment:
Ternary operation can be removed from here

Reply:
I'll correct it